<?xml version="1.0" ?>
<launch>
  <node name="serial_sync" pkg="subact_gripper" type="serialSyncro.py" required="true" output="screen"/>
  <node name="force_meassuring_dev" pkg="subact_gripper" type="gforceMeasDevice.py" required="false" output="screen"/>
</launch>
